docs/edge.md

```diff
@@ -1,92 +1,141 @@
+
+!!! Example
+    In a section where a strategy made three transactions $O = \{3.5, -1, 15, 0\}$:<br>
+    $T_{win} = \{3.5, 15\}$<br>
+    $T_{lose} = \{-1, 0\}$<br>
+
+### Win Rate and Lose Rate
+
+The win rate $W$ is the proportion of winning trades with respect to all the trades made by a strategy. We use the following function to compute the win rate:
+
+$$W = \frac{\sum^{o \in T_{win}} o}{N}$$
+
+Where $W$ is the win rate, $N$ is the number of trades and, $T_{win}$ is the set of all trades where the strategy made money.
+
+Similarly, we can compute the rate of losing trades:
+
+$$ 
+    L = \frac{\sum^{o \in T_{lose}} o}{N} 
+$$
+
+Where $L$ is the lose rate, $N$ is the amount of trades made and, $T_{lose}$ is the set of all trades where the strategy lost money. Note that the above formula is the same as calculating $L = 1 – W$ or $W = 1 – L$
 
 ### Risk Reward Ratio
 
-Risk Reward Ratio (*R*) is a formula used to measure the expected gains of a given investment against the risk of loss. It is basically what you potentially win divided by what you potentially lose:
+Risk Reward Ratio (*R*) is a formula used to measure the expected gains of a given investment against the risk of loss. It is basically what you potentially win divided by what you potentially lose. Formally:
 
-```
-R = Profit / Loss
-```
+$$ R = \frac{\text{potential_profit}}{\text{potential_loss}} $$
 
-Over time, on many trades, you can calculate your risk reward by dividing your average profit on winning trades by your average loss on losing trades:
+??? Example "Worked example of $R$ calculation"
+    Let's say that you think that the price of *stonecoin* today is $10.0. You believe that, because they will start mining stonecoin it will go up to $15.0 tomorrow. There is the risk that the stone is too hard, and the GPUs can't mine it, so the price might go to $0 tomorrow. You are planning to invest $100.<br>
+    Your potential profit is calculated as:<br>
+    $\begin{aligned} 
+        \text{potential_profit} &= (\text{potential_price} - \text{cost_per_unit}) * \frac{\text{investment}}{\text{cost_per_unit}} \\
+                                &= (15 - 10) * \frac{100}{15}\\
+                                &= 33.33
+    \end{aligned}$<br>
+    Since the price might go to $0, the $100 dolars invested could turn into 0. We can compute the Risk Reward Ratio as follows:<br>
+    $\begin{aligned}
+        R   &= \frac{\text{potential_profit}}{\text{potential_loss}}\\
+            &= \frac{33.33}{100}\\
+            &= 0.333...
+    \end{aligned}$<br>
+    What it effectivelly means is that the strategy have the potential to make $0.33 for each $1 invested. 
 
-```
-Average profit = (Sum of profits) / (Number of winning trades)
+On a long horizonte, that is, on many trades, we can calculate the risk reward by dividing the strategy' average profit on winning trades by the strategy' average loss on losing trades. We can calculate the average profit, $\mu_{win}$, as follows:
```

Comment:
```suggestion
On a long horizon, that is, on many trades, we can calculate the risk reward by dividing the strategy' average profit on winning trades by the strategy' average loss on losing trades. We can calculate the average profit, $\mu_{win}$, as follows:
```

horizonte is the right word - but i don't think it's english, but spanish/italian ...